### DISCUSSION TITLE
Laravel Framework

### DISCUSSION THREAD

**Discussion by mightysai1997** [PHPCSStandards/PHP_CodeSniffer#514] [Q&A]
Hi ,
     Is that PHP_CS will detect vulnerabilities in the Laravel Framework ?

**Comment by rodrigoprimo** [comment]
Hi, @mightysai1997!

I'm not sure I understand your question. Could you please provide more details about what you are looking for? I might be missing something, as I don't know much about the Lavarel framework. Which standard are you using or plan to use?

Generally speaking, PHPCS sniffs can detect some types of security vulnerabilities in PHP code. Which vulnerabilities are detected depends on which sniffs are used in a given standard. PHPCS includes a few standards, but there are also several custom standards, and those can also include custom sniffs to check for security vulnerabilities or other things.

For example, the [WordPress Coding Standards](https://github.com/WordPress/WordPress-Coding-Standards) is a custom PHPCS standard that [includes some security-related sniffs](https://github.com/WordPress/WordPress-Coding-Standards/tree/develop/WordPress/Sniffs/Security).
